MSAL/src/native_auth/network/errors/reset_password/MSALNativeAuthResetPasswordPollCompletionResponseError.swift

```diff
@@ -60,6 +60,8 @@ extension MSALNativeAuthResetPasswordPollCompletionResponseError {
              .invalidRequest,
              .userNotFound:
             return .init(type: .generalError, message: errorDescription)
+        case .none:
```

Comment:
Can this not be part of the previous `case`?

Reply:
It can as it's the same, but wanted feedback on how to address the `.none` cases, if they should be grouped or not with the rest

Reply:
@nilo-ms @diegojerezba you ok to have it in the same case also?

Reply:
Yeah, I agree. Can you please check also other switch cases where you use ".none" case?

Reply:
Done for all `.none` cases